suite-native/module-device-settings/src/components/DevicePinProtectionCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { useSelector } from 'react-redux';
+
+import { selectDevice, selectIsDeviceProtectedByPin } from '@suite-common/wallet-core';
+import { Box, HStack, Text, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+
+import { DevicePinActionButton } from './DevicePinActionButton';
+import { DeviceSettingsCard } from './DeviceSettingsCard';
+
+export const DevicePinProtectionCard = () => {
+    const device = useSelector(selectDevice);
+    const isDeviceProtectedByPin = useSelector(selectIsDeviceProtectedByPin);
+
+    if (!device) {
+        return;
+    }
+
+    return (
+        <DeviceSettingsCard
+            icon="password"
+            title={<Translation id="moduleDeviceSettings.pinProtection.title" />}
+        >
+            <VStack marginTop="sp2" spacing="sp16">
```

Comment:
Right padding seems to be off, it supposed to be 16, same as on the bottom but it's less.
![image](https://github.com/user-attachments/assets/d5f0abda-ca43-4464-80bf-6457beed68e1)

Reply:
This fixes the issue for me 06abe1fa9d5ff106ac8a2b6692d52e69ca8ad031